great_expectations/datasource/fluent/config.py

```diff
@@ -349,7 +349,7 @@ def _convert_fluent_datasources_loaded_from_yaml_to_internal_object_representati
     config: Dict[str, Any], _allow_empty: bool = False
 ) -> Dict[str, Any]:
     if _FLUENT_DATASOURCES_KEY in config:
-        fluent_datasources: dict = config[_FLUENT_DATASOURCES_KEY]
+        fluent_datasources: dict = config[_FLUENT_DATASOURCES_KEY] or {}
```

Comment:
@Kilo59 This is needed, because `great_expectations.yml` may come with an empty `fluent_datasources` section.